src/main/scala/org/allenai/pipeline/Workflow.scala

```diff
@@ -72,14 +73,14 @@ case class Workflow(nodes: Map[String, Node], links: Iterable[Link], title: Stri
           textLines.mkString("<ul><li>", "</li><li>", "</li></ul>")
         }
         s"""        g.setNode("$id", {
-                                    |       class: "$clazz",
-                                                            |       labelType: "html",
-                                                            |       label: generateStepContent(${name.toJson},
-                                                                                                               |         ${desc.toJson},
-                                                                                                                                         |         ${info.executionInfo.toJson},
-                                                                                                                                                                                 |         ${params.toJson},
-                                                                                                                                                                                                             |         ${linksJson})
-                                                                                                                                                                                                                                     |     });""".stripMargin
+                                    |      class: "$clazz",
+                                                           |      labelType: "html",
+                                                           |      label: generateStepContent(${name.toJson},
+                                                                                                             |        ${desc.toJson},
+                                                                                                                                      |        ${info.executionInfo.toJson},
+                                                                                                                                                                             |        ${params.toJson},
+                                                                                                                                                                                                        |        ${linksJson})
+                                                                                                                                                                                                                               |    });""".stripMargin
```

Comment:
`sbt format` always does weird things to this code block. I've cleaned it up, but the formatter puts it in this state.